src/lwc/utilPageLevelMessage/utilPageLevelMessage.js

```diff
@@ -0,0 +1,74 @@
+import { LightningElement, api } from 'lwc';
+
+const SUCCESS = 'success';
+const WARNING = 'warning';
+const ERROR = 'error';
+const INFO = 'info';
+const THEME_SUCCESS = 'slds-theme_success';
+const THEME_WARNING = 'slds-theme_warning';
+const THEME_ERROR = 'slds-theme_error';
+const THEME_INFO = 'slds-theme_info';
+const TEXT_ERROR = 'slds-text-color_error';
+
+export default class utilPageLevelMessage extends LightningElement {
+    @api title;
+    @api subtitle;
+    @api errors;
+    @api iconName;
+    @api iconSize;
+    @api iconVariant;
+    @api iconDescription;
+    @api variant;
+
+    get titleText() {
+        return this.title ? this.title : 'Review the errors on this page.';
+    }
+
+    get subtitleText() {
+        return this.subtitle ? this.subtitle : '';
+    }
+
+    get hasIconName() {
+        return this.iconName ? true : false;
+    }
+
+    get assistiveText() {
```

Comment:
At first glance this looks odd, but are we looking to eventually return something other than SUCCESS for SUCCESS and so on in the future? Maybe we can set up an assistive text map variable?